### PR TITLE
Download token images

### PR DESCRIPTION
**Pull request by anxolin**
Adds a simple script to download the images from the file, and adds them to the project.

It can be used just to make sure the images can be downloaded, but the main reason why this was done now, is just to add all the images to the project to then upload to IPFS automatically and update the `imageUris` to use IPFS.

Run it by executing:
```
yarn downloadImages
```

image

### CODE REVIEW

File: src/downloadImages.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { fileURLToPath } from 'url';
+
+
+
+async function main() {
+  for(const token of cowSwapList.tokens) {
+    const { symbol, name, address, chainId, logoURI } = token
+    const filePath = `public/images/${chainId}/${address}.png`
+
+    console.log(`Download image for ${symbol} (${name})
```
Comment (fleupold) [inline src/downloadImages.js:14]: Maybe only fetch if the file doesn't yet exist. If we want to change the image, feels reasonable to delete the old one first?

Comment (anxolin) [reply src/downloadImages.js:14]: sure, done. This is not meant to be executed a lot after we define a good automatic process to add tokens.

here is a test where I just deleted 3 images to download again
image

**Comment by anxolin** [comment]
> Would be great if this could also resize images to the desired quality.

Yes, but if we do this Is in other PR. I think we should go step by step:
- download images
- upload to ipfs
- update the JSON with the new images
- publish to s3
- GH action for new releases
- Optimize images

> Also inside the cowswap list we will still reference the old images. Will there be another script that creates a list with the locally hosted ones?

Yes, it would be other, i actually started and is easy to finish. But Im putting these parts on hold after what we discussed
https://github.com/cowprotocol/token-lists/pull/11

**Comment by fleupold** [comment]
> Yes, but if we do this Is in other PR. I think we should go step by step:

💯 agree! I think this would be a great automation task for @ahhda to work on once the DevOps migration is finished. For now (ie the next 1-2 months) I think having the manual process is fine.

**Review by alfetopito — COMMENTED** [review]
👍

**Pull request merged**